bin/uno

```diff
@@ -67,13 +67,18 @@ setup)
   fluo-only)
     "$bin"/impl/setup-fluo-only.sh "${@:2}"
     ;;
+  metrics)
+    export SETUP_METRICS=true
+    "$bin"/impl/setup-metrics.sh "${@:2}"
+    ;;
   *)
     echo "Usage: uno setup <command>"
     echo -e "\nPossible commands:\n"
     echo "    accumulo   Sets up Accumulo along with Hadoop, Zookeeper and Spark"
     echo "    fluo       Sets up Fluo along with Accumulo (and its dependences)."
     echo "               Also, sets up metrics server (if SETUP_METRICS=true in env.sh)"
     echo "    fluo-only  Sets up Fluo only. Useful for upgrading Fluo without losing cluster data."
+    echo "    metrics    Sets up metrics service (InfluxDB + Grafana)."
```

Comment:
Why not update the readme with info about this command?  Would be useful to add something to readme about how this can be used with mini fluo.